test_util/azure_test_driver.py

```diff
@@ -163,7 +162,7 @@ def poll_deploy():
 
         print('Detected IP configuration: {}'.format(ip_buckets))
 
-        with closing(SSHTunnel(get_value('linuxAdminUsername'), 'ssh_key', master_lb, port=2200)) as t:
+        with SSHTunnel(get_value('linuxAdminUsername'), 'ssh_key', master_lb, port=2200) as t:
```

Comment:
can we pull the closing() -> __enter__  + __exit__ into a separate very simple quick to land PR?